go.mod

```diff
@@ -105,9 +105,10 @@ require (
 	github.com/pierrec/lz4 v2.6.0+incompatible
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/dynamiclistener v0.2.3
+	github.com/rancher/lasso v0.0.0-20210616224652-fc3ebd901c08
 	github.com/rancher/remotedialer v0.2.0
 	github.com/rancher/wharfie v0.3.4
-	github.com/rancher/wrangler v0.6.2
+	github.com/rancher/wrangler v0.8.5
 	github.com/rancher/wrangler-api v0.6.0
```

Comment:
Hmm, something is still using old wrangler if this line doesn't get removed by `go mod tidy`.

Reply:
pretty sure this is solved now